pygeo/parameterization/DVGeo.py

```diff
@@ -1858,14 +1890,22 @@ def convertSensitivityToDict(self, dIdx, out1D=False):
 
         # Add in child portion
         for iChild in range(len(self.children)):
-            childdIdx = self.children[iChild].convertSensitivityToDict(dIdx, out1D=out1D)
+            childdIdx = self.children[iChild].convertSensitivityToDict(dIdx, out1D=out1D, compName=compName)
             # update the total sensitivities with the derivatives from the child
             for key in childdIdx:
                 if key in dIdxDict.keys():
                     dIdxDict[key] += childdIdx[key]
                 else:
                     dIdxDict[key] = childdIdx[key]
 
+        # replace other names with user
+        if compName and self.useComposite:
```

Comment:
Related: Is there a situation where `self.useComposite` is True, but `compName` is None?

Reply:
Yeah I agree, I'm not sure what's a good name but can definitely change that.

compName here is a boolean, and it can be False when useComposite is True, since we need to compute sensitivities both ways.